api/test/wfl/system/automation_test.clj

```diff
@@ -1,8 +1,8 @@
 (ns wfl.system.automation-test
-  (:require [clojure.pprint        :as pprint]
-            [clojure.test          :refer [deftest]]
+  (:require [clojure.test          :refer [deftest is]]
             [wfl.tools.fixtures    :as fixtures]
             [wfl.tools.resources   :as resources]
+            [wfl.service.cromwell  :refer [final-statuses]]
```

Comment:
We may want to define a new set of `active-statuses` for Firecloud workflows as it turns out they are different from the Cromwell's: `[ Queued, Launching, Submitted, Running, Failed, Succeeded, Aborting, Aborted, Unknown ]`?

Reply:
Created https://broadinstitute.atlassian.net/browse/GH-1410